patterns/product-hero.php

```diff
@@ -4,43 +4,43 @@
  * Slug: woocommerce-blocks/product-hero
  * Categories: WooCommerce
  */
+
+$query = new \WC_Product_Query(
+	array(
+		'limit'  => 1,
+		'return' => 'ids',
+		'status' => array( 'publish' ),
+	)
+);
+
+$products   = $query->get_products();
+$product_id = $products ? $products[0] : null;
 ?>
-<!-- wp:group {"align":"wide","style":{"color":{"background":"#6b7ba8"},"spacing":{"blockGap":"0","padding":{"top":"1em","right":"1em","bottom":"1em","left":"1em"}}},"textColor":"background","layout":{"type":"constrained","contentSize":"100%","justifyContent":"left"}} -->
-<div class="wp-block-group alignwide has-background-color has-text-color has-background" style="background-color:#6b7ba8;padding-top:1em;padding-right:1em;padding-bottom:1em;padding-left:1em">
-	<!-- wp:columns -->
-	<div class="wp-block-columns">
-		<!-- wp:column {"width":"40%"} -->
-		<div class="wp-block-column" style="flex-basis:40%"><!-- wp:image {"id":1,"sizeSlug":"full","linkDestination":"none"} -->
-			<figure class="wp-block-image size-full"><img src="<?php echo esc_url( plugins_url( 'images/pattern-placeholders/leather-guitar-typewriter-red-gadget-sofa.png', dirname( __FILE__ ) ) ); ?>" alt="<?php esc_attr_e( 'Placeholder image used to represent a product being showcased in a hero section.', 'woo-gutenberg-products-block' ); ?>" class="wp-image-1"/></figure>
-			<!-- /wp:image --></div>
+
+<!-- wp:woocommerce/single-product {"productId":<?php echo esc_attr( $product_id ); ?>,"align":"wide"} -->
+<div class="wp-block-woocommerce-single-product alignwide">
+	<!-- wp:columns {"style":{"color":{"background":"#6b7ba8"},"elements":{"link":{"color":{"text":"var:preset|color|base"}}},"spacing":{"padding":{"left":"20px"}}},"textColor":"base"} -->
+	<div class="wp-block-columns has-base-color has-text-color has-background has-link-color" style="background-color:#6b7ba8;padding-left:20px">
+		<!-- wp:column {"width":"40%","style":{"spacing":{"padding":{"top":"0","right":"0","bottom":"0","left":"0"}}}} -->
+		<div class="wp-block-column" style="padding-top:0;padding-right:0;padding-bottom:0;padding-left:0;flex-basis:40%">
+			<!-- wp:post-featured-image {"height":"300px"} /-->
```

Comment:
Instead of using the post's featured image, how about using the Product Image block (as we have within the single product block?)

Reply:
I first tried to use the `Product Image` block, but it does not have image size settings available, so I could not achieve the design. Do you think it's a problem using the `Post Featured Image`?

Reply:
Yeah, I'm not sure about the `Product Image` block either.
I'll create an issue to add the size settings to it and replace the image in this pattern, so we can go ahead with this PR but remember to switch blocks when possible.